dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/MetadataDumper.java

```diff
@@ -300,6 +335,13 @@ public <T> T runChildTask(Task<T> task) throws MetadataDumperUsageException {
     }
   }
 
+  private int countTasks(List<Task<?>> tasks) {
+    return tasks.stream()
+        .map(task -> task instanceof TaskGroup ? countTasks(((TaskGroup) task).getTasks()) : 1)
+        .reduce(Integer::sum)
```

Comment:
If you changed `map` to `mapToInt`, then you could take advantage of the existing `sum` method.

https://docs.oracle.com/javase/8/docs/api/java/util/stream/IntStream.html#sum--